### ISSUE TITLE
Can't prepare filestore

### ISSUE THREAD

**Issue by sonofmun** [OpenPhilology/nidaba#8]
When I use the following command 

    nidaba batch -b nlbin:threshold=0.5,zoom=0.5,escale=1.0,border=0.1,perc=80,range=20,low=5,high=90 -l tesseract -o kraken:model=greek -p spell_check:dict=polytonic_greek -- /home/mmunson/phaistos/OCR/in_progress/OCR/hvd.swete.1.1901.kraken/*.png

I get the following response

    Preparing filestore             [✗]

And then the job exits.  The permissions on the the destination folder (phaistos/OCR/in-progress/OCR) are 777, so it doesn't appear to be a permissions problem.

**Comment by sonofmun** [comment]
I updated nidaba again and now it can prepare the filestore.  I had done 

    pip install -U kraken

after the last time I updated nidaba and it appears to have installed some older versions of libraries that may have messed up the creation of the filestore.  When I upgraded nidaba, it upgraded the following packages:

Pillow-3.0.0 from 2.8.1
argparse-1.2.1 
billiard-3.3.0.21 from 3.3.0.20
celery-3.1.19 from 3.1.18
click-5.1 from 4.0
kombu-3.0.29 from 3.0.28
numpy-1.10.1 from 1.9.2
regex-2015.10.22 from 2015.3.18

I don't know if any of these newer packages will cause a problem with Kraken.
